src/utils/filter-utils.js

```diff
@@ -0,0 +1,51 @@
+const sortLowToHigh = (items) => {
+    return [...items.sort((first, second) => first.price - second.price)];
+};
+
+const sortHighToLow = (items) => {
+    return [...items.sort((first, second) => second.price - first.price)];
+};
+
+const sortItems = (items, sortType) => {
+    switch (sortType) {
+        case "lowToHigh":
+            return sortLowToHigh(items);
+        case "highToLow":
+            return sortHighToLow(items);
+        default:
+            return items;
+    }
+};
+
+const filterPriceRange = (items, filterPrice) => {
+    return items.filter((item) => {
+        return parseInt(item.price) <= filterPrice;
+    });
+};
+
+const filterByRating = (items, rating) => {
```

Comment:
you have to make another separate file for array manipulation operation  so it will look clean